testsuite/tests/tool-toplevel/error_highlighting.compilers.reference

```diff
@@ -1,4 +1,4 @@
-Characters 34-41:
+Characters 197-204:
```

Comment:
This is caused by the additional text in the ocamltest comment. I assume further locations do not change because the toplevel resets the location counter before each phrase.